src/analyze.cpp

```diff
@@ -7266,8 +7273,6 @@ static void resolve_llvm_types(CodeGen *g, ZigType *type, ResolveStatus wanted_r
 
 LLVMTypeRef get_llvm_type(CodeGen *g, ZigType *type) {
     assertNoError(type_resolve(g, type, ResolveStatusLLVMFull));
-    assert(type->abi_size == 0 || type->abi_size == LLVMABISizeOfType(g->target_data_ref, type->llvm_type));
```

Comment:
This assertion can stay, right?

Reply:
I don't think this holds anymore, because a larger alignment required for a field can increase the size of a struct.

It would be possible to assert
```
assert(type->abi_size == 0 || type->abi_size >= LLVMABISizeOfType(g->target_data_ref, type->llvm_type));
```
instead.

Reply:
Ah that makes sense. Here's another test case to add:

```zig
comptime {
    expect(@sizeOf(extern struct {
        x: u128,
        y: u8,
    }) == 32);
} 
```

Reply:
Done.